packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -85,17 +85,24 @@ export const PaymentErrorView = ({
     <>
       {title}
       <section
-        className={`container card payment-error ${className}`}
+        className={`container card payment-error row-start-3 row-end-4 mt-2 tablet:mt-0 ${className}`}
         data-testid="payment-error"
       >
-        <div className="wrapper">
-          <img id="error-icon" src={errorIcon} alt="error icon" />
+        <div className="wrapper mb-16">
+          <img
+            className="mt-16 mb-10 mx-auto"
+            src={errorIcon}
+            alt="error icon"
```

Comment:
nit: Should this be translated?
Obvs not a blocker since this wasn't translated earlier.

Reply:
Nice catch! It's also an a11y pain to have that type of `alt` text. I opted to remove the text altogether and leave the `alt` empty. The title above and description below explain that there is an error and why, making the image more decorative than informative in this case.

![payment_error_page](https://user-images.githubusercontent.com/104124653/181862815-99db54e0-fe24-48be-8068-1fb7c4bd8ae9.jpg)

References:
- https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt
- https://www.w3.org/WAI/tutorials/images/decorative/
- https://www.portent.com/blog/design-dev/decorative-images-how-to-identify-and-tag-them-for-accessibility.htm
- https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices

Reply:
Aligned, thanks for the reference! :)